misc/skins/Makefile.am

```diff
@@ -1,17 +1,23 @@
 skindir = $(pkgdatadir)/skins
 
 skin_DATA = \
+	catppuccin.ini \
```

Comment:
By convention, all our 256-color skins are named `something256.ini` to make it possible to have fallback non-256 versions and possibly boosted 16M versions of the same general scheme.

I think it would make sense to stick to this convention, and I would appreciate you renaming the files.